src/mds/chunkserverclient/chunkserver_client.cpp

```diff
@@ -48,6 +48,32 @@ namespace curve {
 namespace mds {
 namespace chunkserverclient {
 
+void ChunkServerClient::printErrorCode(int errorCode){
```

Comment:
Hi, you shouldn't print error directly in here.

What we need it a function that can transfer error code to corresponding name like https://github.com/NopeDl/curve/blob/0a8642a53c801fc3f29c4e5ebdd5bb304a1e9301/src/client/client_common.h#L241, and when we encounter an error, we can use that function to print its name.

BTW, there are a number of status codes that need to be modified and adjusted, and you can start by doing a general search and listing them.